library/std/src/sys/windows/process.rs

```diff
@@ -94,6 +101,13 @@ struct DropGuard<'a> {
     lock: &'a Mutex,
 }
 
+#[derive(Default)]
+struct ProcAttributeList {
+    buf: Vec<u8>,
```

Comment:
This isn't explicitly mentioned in the Windows API documentation, but are there any alignment requirements for the `ProcAttributeList`? I would expect at least 8 or 16 byte alignment since that is what `malloc` gives you in C.

Reply:
If `PROC_THREAD_ATTRIBUTE_LIST` were to be defined as an extern type, then this would be a great example of allowing alignment on extern types. I'm not aware of any alignment requirements, but 8/16 byte alignment would be a safe assumption (as that is what `HeapAlloc` guarantees).